src/components/SignUp/SignUp.js

```diff
@@ -8,7 +8,15 @@ export default class SignUp extends Component {
     name: '',
     login: '',
     password: '',
-    repeatPasswod: ''
+    repeatPasswod: '',
```

Comment:
improvements to consider: have a single object for errors with field name as key and error message as value. treat undefined, null or empty values as no error